src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityCreateConnectionRequestController.php

```diff
@@ -68,15 +69,59 @@ public function connectionRequestFromEntityAction(
         $form->handleRequest($request);
 
         if ($form->isSubmitted()) {
-            if ($this->isCancelAction($form)) {
-                return $this->returnToOverview($serviceId);
+            $entity = $this->entityService->getManageEntityById($manageId, $environment);
+            $parameters = ['entityName' => $entity->getMetaData()->getNameEn()];
+
+            if ($this->isCancelAction($form) || !count($command->getConnectionRequests())) {
```

Comment:
Nitpick, place the `count` construction in a simple `has` method on the command?